app/models/cautionary_alert.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class CautionaryAlert
+  include ActiveModel::Model
+
+  attr_accessor :propertyReference, :alerts
+
+  class << self
+    def for_property_reference(reference)
+      response = PlatformApis::CautionaryAlerts::Client.
+        get_cautionary_alerts_by_property_reference(reference)
+
+      response.key?(:message) ? [] : build(response)
+    end
+
+    def build(attributes)
+      new(
+        propertyReference: attributes["propertyReference"],
+        # rubocop:disable Layout/LineLength
+        alerts: attributes["alerts"].empty? ? {} : Alert.build(attributes["alerts"].first)
```

Comment:
updated in https://github.com/LBHackney-IT/repairs-api/pull/14/commits/bda8c2bcbfb3fe24bf2051c6e29ff55a9768da95